sklearn/covariance/graph_lasso_.py

```diff
@@ -21,7 +21,8 @@
 from ..utils.validation import check_random_state, check_array
 from ..linear_model import lars_path
 from ..linear_model import cd_fast
-from ..cross_validation import _check_cv as check_cv, cross_val_score
+from ..model_selection.partition import _check_cv as check_cv
+from ..model_selection.validation import cross_val_score
```

Comment:
Yes it is available at top level `model_selection` but like you mentioned in the other comment should we not use the more expanded one for internal use?

This however needs to be changed for docstring examples.